live-examples/js-examples/string/string-raw.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">const name = 'Bob';
+
+// The next line returns the template with the
+// value of the name variable inserted.
+console.log(string.raw`<p>Good morning, {name}</p>`);
```

Comment:
This logs "Error: string is not defined" - I think it needs `String` not `string`. Also it would need to be ${name} for the substitution to work.

Reply:
But also, I'm not sure this is an ideal example of `raw`. With this string, just console.log gives exactly the same output:

```
console.log(`<p>Good morning, ${name}</p>`);
```

(I'm also not sure what the `<p>`/`</p>` tags are for here)

As I understand it, the point of `raw` is that it doesn't process escapes:

```
console.log(String.raw`Good morning,\n ${name}`);
// expected output: "Good morning,\n Bob"

console.log(`Good morning,\n ${name}`);
// expected output: "Good morning,
//                  Bob"
```